packages/dev/create-react-app/templates/default/src/index.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
```

Comment:
I think React is trying to move everyone to `import * as React` (same for react-dom)? Also Flow only work correctly with the namespace import